fast-af/fast-checkout-woocommerce
# PINT-387 Add an option to hide the login button in the footer

# Description

By default, the Fast Login button is rendered in the footer. This update provides an option to hide the login button in the footer in favor of loading it through a widget (See #28).

# Testing instructions

1. Login to the [WordPress admin on the staging site](https://fasttestdev.wpcomstaging.com/wp-admin/).
2. Go to the [Fast settings page](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast).
3. Uncheck the "Display Fast Login Button in Footer" option.
4. Save the settings.
5. Verify that the Fast Login button is not displayed in the footer when it should be.

# Checks
